FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -633,8 +632,7 @@ def export_getComponentDocumentation( self, cType, system, module ):
     except Exception as _e:
       return S_ERROR( 'No documentation was found' )
 
-  types_storeHostInfo = []
-  def export_storeHostInfo( self ):
+  def __storeHostInfo( self ):
```

Comment:
And then of course readHostInfo needs to be staticmethod as well, but that function doesn't use self so it is OK to change that...